src/qt/bitcoingui.cpp

```diff
@@ -113,7 +113,13 @@ BitcoinGUI::BitcoinGUI(const NetworkStyle *networkStyle, QWidget *parent) :
 #endif // ENABLE_WALLET
     if(enableWallet)
     {
-        windowTitle += tr("Wallet");
+        QString userWindowTitle = QString::fromStdString(GetArg("-windowtitle", ""));
+        if(userWindowTitle.isEmpty()){
+            windowTitle += tr("Wallet");
```

Comment:
That was used for indicating a mode - "Wallet" or "Node". How about keeping it and adding userWindowTitle after instead? I.e.
```
if(enableWallet)
{
    windowTitle += tr("Wallet");
} else {
    windowTitle += tr("Node");
}
QString userWindowTitle = QString::fromStdString(GetArg("-windowtitle", ""));
if(!userWindowTitle.isEmpty()) windowTitle += " - " + userWindowTitle;
```